doc/rst/source/GMT_Docs.rst

```diff
@@ -6079,7 +6083,7 @@ by modifying the :ref:`IO_HEADER_MARKER <IO_HEADER_MARKER>` default setting.
 Fields within a record must be separated by
 spaces, tabs, commas, or semi-colons. Each field can be an integer or floating-point
 number or a geographic coordinate string using the
-[±]dd[:mm[:ss]][W:\ \|\ S\ \|\ N\ \|\ E\ \|\ w\ \|\ s\ \|\ n\ \|\ e]
+[±]*dd*\ [:*mm*\ [:*ss*\ [.*xx...*\ ]]][**W**\ \|\ **E**\ \|\ **S**\ \|\ **N**\ \|\ **w**\ \|\ **e**\ \|\ **s**\ \|\ **n**\ ]
```

Comment:
Not working. 
![image](https://user-images.githubusercontent.com/3974108/55677897-921e4a80-58be-11e9-8da1-33560a225079.png)